controller/auth.js

```diff
@@ -88,15 +90,89 @@ async function login(req, res) {
   throw new errors.FcError(errors.EMAIL_USERNAME_PASSWORD_INVALID);
 }
 
-async function signup(req, res) {
-  const { email, username, full_name, school_name, password } = req.body;
+async function sendOtp(req, res) {
+  if (!req.body.email) {
+    throw new errors.FcError(errors.MISSING_EMAIL);
+  }
+
+  if (await models.User.findOne({where: { email: req.body.email }}) ) {
+    throw new errors.FcError(errors.EMAIL_EXISTS);
+  }
+
+
+  const otp = otpGenerator.generate(6, {
```

Comment:
Is this package handle duplicate case ?

Reply:
You mean duplicate otp codes?

Reply:
Yeah

Reply:
No. Yet i think otp don't need to be unique.